build.gradle

```diff
@@ -96,6 +99,12 @@ task integration(type: Test) {
   failFast = true
 }
 
+task fortifyScan(type: JavaExec)  {
+  main = "uk.gov.hmcts.fortifyclient.FortifyClientMainApp"
+  classpath += sourceSets.test.runtimeClasspath
+  jvmArgs = ['--add-opens=java.base/java.lang.reflect=ALL-UNNAMED']
+}
```

Comment:
We can add the below flag to prevent the build from failing if the Fortify scan detects issues as we monitor the fortify app to fix/suppress the issues.
ignoreExitValue = true

Reply:
done